src/test/resources/features/DataInput.feature

```diff
@@ -1,7 +1,7 @@
 Feature: HOCS User is add data to a case
 
   Background:
-    Given I am user "DCU"
+    Given I am user "DANNY"
```

Comment:
This is probably better being changed to a official user type that would log into the system.

Reply:
Permissions and Admin UI are not yet complete, this is currently one of two users with the required permissions available to run the tests.  Please resolve.

Reply:
No problem.